pkg/disttask/importinto/encode_and_sort_operator_test.go

```diff
@@ -185,6 +185,12 @@ func TestGetWriterMemorySizeLimit(t *testing.T) {
 			dataKVMemSizePerCon:     460175067,
 			perIndexKVMemSizePerCon: 153391689,
 		},
+		{
+			createSQL:               "create table t (a int, b int, c int, primary key(a,b,c) clustered, key(b,c), unique(b), unique(c), key(a,b))",
+			numOfIndexGenKV:         4,
+			dataKVMemSizePerCon:     460175067,
+			perIndexKVMemSizePerCon: 153391689,
+		},
 		{
 			createSQL:               "create table t (a int, b int, c int, primary key(a,b,c) nonclustered, key(b,c), unique(b), unique(c), key(a,b))",
```

Comment:
Do it cover the `ClusteredIndexDefModeIntOnly`? check with that option and a table with non-int PK

Reply:
> // ClusteredIndexDefModeIntOnly indicates only single int primary key will default be clustered.

seems not related, composite int pk is also CommonHandle

Reply:
OK I see that `HasClusteredIndex` can both check `t.PKIsHandle || t.IsCommonHandle`